spec/identity/hostdata_spec.rb

```diff
@@ -38,6 +38,19 @@
       end
     end
 
+    context 'when the LOGIN_DOMAIN env var is set' do
+      before do
+        stub_const(
+          'ENV',
+          { 'LOGIN_DOMAIN' => 'identityenvbox.gov', 'LOGIN_DATACENTER' => 'true' },
+        )
```

Comment:
consider doing a `stub_const('ENV', env)` once at the top of the file? and then just `let(:env) { { 'LOGIN_DOMAIN' => 'aaaa' } }`